openlibrary/templates/type/edition/view.html

```diff
@@ -168,7 +168,8 @@
 $if ebook_status == "read-online":
```

Comment:
I don't think there is any way to have a book readable at OL without some ocaid. That's literally the mechanism that enables the "read online" function.